action-mongo-atlas-nuke.py

```diff
@@ -44,15 +41,16 @@ def delete_project(public: str, private: str, url: str, project_id: str) -> None
     except Exception as error:
         print(error)
         raise
```

Comment:
I'm not sure what the need here is to catch an exception and then throw another one? You might instead check for specific `status_code` values like 404.